src/services/chat/prepopulated.js

```diff
@@ -41,15 +41,29 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser.defaulValues
+                  updated_at: ownUser?.updated_at,
+                  is_anonymous: ownUser?.is_anonymous,
+                  name: ownUser?.username,
+                  image: ownUser?.profile_pic_path,
+                  last_active: ownUser?.last_active_at,
+                  username: ownUser?.username,
+                  id: ownUser?.user_id,
+                  created_at: ownUser?.created_at
                 },
                 user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_anonymous` property is hardcoded to `false`. If the `ownUser` object has an `is_anonymous` property, it would be better to use that instead of hardcoding the value.
```diff
- is_anonymous: false,
+ is_anonymous: ownUser?.is_anonymous,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->